### ISSUE TITLE
Non-functional masked dice roll

### ISSUE THREAD

**Issue by keldrilh** [Miskatonic-Investigative-Society/CoC7-FoundryVTT#439]
Hello,
when I ask my players to blind dice, they can see it.
Thanks and have a nice day

**Comment by HavlockV** [comment]
Hi keldrilh, thanks for the feedback. I just tried and couldn't reproduce this. Could you give a little bit more details and eventually a screen shot ?

**Comment by mart-pilon** [comment]
Hello,

I have the same issue when using Dice so nice with CoC, if I select Self Roll (Jet Privé in French), while the roll doesn't shows up in the chat, the dices are still showing up! I tried adding modules like Hidden GM Rolls and Actually Private Rolls with no avail.

I made sure it wasn't a module conflict by disabling all modules (except DsN) and I also tested both French and English (as I play in French) to eliminate that possibility as well.

Here's a screenshot from the player side.

Thanks!

![image](https://user-images.githubusercontent.com/84545977/119055748-92430c80-b997-11eb-85c2-d73ce53c9e65.png)
